hadoop-hdds/framework/src/main/resources/webapps/static/ozone.js

```diff
@@ -48,8 +48,13 @@
   });
   angular.module('ozone').component('jvmParameters', {
     templateUrl: 'static/templates/jvm.html',
-    controller: function($http) {
+    controller: function($http, $scope) {
       var ctrl = this;
+
+      $scope.toggleContent = function() {
+        $scope.contentVisible = !$scope.contentVisible;
```

Comment:
Just wanna check what is the default value of `$scope.contentVisible`? And where was it initialized?

Reply:
The initial value of `$scope.contentVisible` should be false, as this variable controls whether the JVM page expands to show detailed configuration. I believe we should assign an initial value to this variable to make the code more standardized.